src/components/src/VRProductDisplayItem/VRProductDisplayItem.tsx

```diff
@@ -66,24 +190,26 @@ class VRProductDisplayItem extends Component<IVRComponentProps, IVRComponentStat
         <div className="vr-window-container">
           <button type="button" className="exit-btn" onClick={() => handleCloseVR()} />
 
-          <Scene className="vr-container" embedded vr-mode-ui="enterVRButton: #myEnterVRButton;" loading-screen="backgroundColor: #000000; dotsColor: white">
+          <div className="cover-html-texture" />
+
+          {/** The place holder html for description panel.  Replace this div to adjust what is shown! */}
+          <div className="outer-html-texture">
+            <div id="boxHTML" className="inner-html-texture">
+              <p className="html-text-texture">Place holder html</p>
+            </div>
+          </div>
+
+          <Scene className="vr-container" embedded vr-mode-ui="enterVRButton: #myEnterVRButton;" background="color: grey" loading-screen="backgroundColor: #000000; dotsColor: white">
```

Comment:
There's a lot of inline css here. Can we move it to the less file?

Reply:
Not so sure styles will be able to attach attributes to an element.  Those are aframe specific attributes that dictate settings in the scene.

Reply:
Best I can do is maybe format it a little better?

Reply:
I just reformatted it so its easier to read